### PR TITLE
AN-8409 First column of tables should be <th> elements

### PR DESCRIPTION
**Pull request by thallada**
For accessibility reasons, the first column of both the Course List table and the Learner Roster should be cells of <th> elements. This allows the screen reader to prefix the reading of each cell in that row with the content in that first <th> cell to give the data necessary context.

### CODE REVIEW

File: analytics_dashboard/static/apps/course-list/list/views/course-id-and-name-cell.js [hunk 1]
```diff
@@ -13,13 +13,15 @@ define(function(require) {
         CourseIdAndNameCell;
 
     CourseIdAndNameCell = Backgrid.Cell.extend({
+        tagName: 'th',
```
Comment (dsjen) [inline analytics_dashboard/static/apps/course-list/list/views/course-id-and-name-cell.js]: I'm realizing how similar `course-id-and-name-cell.js` and `name-username-cell.js` are.  Can they be refactored?

File: analytics_dashboard/static/sass/_developer.scss [hunk 2]
```diff
@@ -805,6 +805,11 @@ body.view-dashboard {
 
   }
 
+  th.learner-name-username-cell {
+    // override the th styling with td styling
+    @extend td;
```
Comment (dsjen) [inline analytics_dashboard/static/sass/_developer.scss:810]: Nice use of `@extend`!

File: analytics_dashboard/static/apps/course-list/list/views/course-id-and-name-cell.js (hunk 1 above)
Comment (thallada) [reply analytics_dashboard/static/apps/course-list/list/views/course-id-and-name-cell.js]: Good suggestion, I refactored what I could into an abstract RowHeaderCell in the generic-list component.

**Review by dsjen — APPROVED** [review]
Nice refactor!

**Pull request merged**